gini/gini-switch-sdk-ios
# Feedback api

# Introduction

Sending feedback on the received extractions.

# How to test

Run the test app, in the extractions screen, change some field (the text fields are editable now) and see if the backend receives the feedback, or debug to see what's being sent

# Merge info

Automatic
